Gemfile

```diff
@@ -33,6 +33,7 @@ gem 'bootstrap-sass'
 
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
+gem 'simplecov', :require => false, :group => :test
```

Comment:
I would somewhat expect that if you added simplecov .. that you would have reported your code coverage in your PR.  Otherwise this feels somewhat like wasted effort for your challenge response.